src/components/PageContentWrapper/PageContentWrapper.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import React from 'react';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface PageContentWrapperProps extends ComponentProps<'div'> {
+  /**
+   * Contents on a page that are flanked by a Page/Margin in figma
+   */
+  children: React.ReactNode;
+}
+
+/**
+ * ## Overview
+ *
+ * This component wraps content on a page to maintain consistent page padding
+ *
+ * [Figma Link](Add Figma URL here)
```

Comment:
I know there's not a component for this in Figma so could you just link to the Lot Details page and put a comment that this page is wrapped in it?  Thanks!